frank-doc-frontend/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,21 @@
+<header>
+  <div class="title" routerLink="/">
+    <app-icon-ff-logo-small [width]="35" [height]="35"></app-icon-ff-logo-small>
+    <h1>Reference</h1>
+    <span class="version">{{ version }}</span>
+  </div>
+  <nav>
+    <ul>
+      <li><a routerLink="/" routerLinkActive="active" [routerLinkActiveOptions]="{ exact: true }">Home</a></li>
+      <li><a routerLink="/index" routerLinkActive="active">Index</a></li>
+    </ul>
+  </nav>
+  <div class="actions">
+    <button class="switch-theme-button" title="Dark/light Mode">
+      <app-icon-darkmode width="20" height="20" colour="#8A8A8A"></app-icon-darkmode>
```

Comment:
It might be smart to create vars for colors, to make then reusable and easy to change.

Reply:
Sorry, this link is not really explaining what I mend. I mend to do it like this:

```css
body {
  --my-color: #fdc300;
}
```
```html
<component color="var(--my-color)" />

<div [attr.fill]="color" />
```